decidim/decidim
# Fix account update without password change

#### :tophat: What? Why?
As reported at #9567, the user is unable to update their account information without changing their password. This PR fixes that issue.

There was also a problem with the tests as they were testing this situation already but they work differently than for normal users. During tests, the password fields got pre-populated with the correct password because the user record responds to `.password` and `.password_change` during these tests. I also added one more check in the test to check that the encrypted password wasn't changed which indicates a password update.

I also removed the pre-filling from the password fields which means these tests would now also fail without this fix even as they were.

#### :pushpin: Related Issues
- Fixes #9567

#### Testing
- Go to the account form
- Try to do some update without filling the password field
- See the success message on the page
